transforms/images/apply-flatfield-plugin/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "polus-plugins-transforms-images-apply-flatfield"
+version = "2.0.0-dev8"
+description = ""
+authors = [
+    "Nick Schaub <nicholas.schaub@nih.gov>",
+    "Najib Ishaq <najib.ishaq@nih.gov>"
+]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+bfio = { version = "2.1.9", extras = ["all"] }
```

Comment:
Update both bfio and filepattern libraries

Reply:
Done